src/test/java/seedu/mark/logic/commands/ExportCommandTest.java

```diff
@@ -0,0 +1,122 @@
+package seedu.mark.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.mark.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.mark.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.mark.testutil.TypicalBookmarks.getTypicalMark;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.mark.commons.exceptions.DataConversionException;
+import seedu.mark.model.Model;
+import seedu.mark.model.ModelManager;
+import seedu.mark.model.ReadOnlyMark;
+import seedu.mark.model.ReadOnlyUserPrefs;
+import seedu.mark.model.UserPrefs;
+import seedu.mark.storage.Storage;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code ExportCommand}.
+ */
+public class ExportCommandTest {
+    private Model model = new ModelManager(getTypicalMark(), new UserPrefs());
+    private Model expectedModel = new ModelManager(getTypicalMark(), new UserPrefs());
+    private Storage storage = new StorageStubAllowsSave();
+
+    @Test
+    public void execute_validFile_success() {
+        Path filePath = Path.of("data", "myValidFile");
```

Comment:
Is there any difference between `Path.of()` and `Paths.get()`? I saw both methods are used.

Reply:
I'll standardise it to `Path.of()`! `Paths.get()` is used elsewhere in the code base, but it's [not recommended](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/Paths.html) as it may be deprecated in future.